src/libs/apistore.ts

```diff
@@ -1,10 +1,11 @@
 import style from './style.js'
-import {StyleSpecification, format} from '@maplibre/maplibre-gl-style-spec'
+import {format} from '@maplibre/maplibre-gl-style-spec'
+import type {StyleSpecification} from 'maplibre-gl'
 import ReconnectingWebSocket from 'reconnecting-websocket'
 
 export type ApiStyleStoreOptions = {
-  port?: string
-  host?: string
+  port: string | null
```

Comment:
Nitpick, but isn't the current syntax using ? cleaner?

Reply:
It is, but it's also not the same - one is string or null while the other is srting or undefined.
JavaScript is annoying in that aspect...